### PR TITLE
BMO Amnesia

### PR DESCRIPTION
**Pull request by purelogiq**

### CODE REVIEW

**Comment by purelogiq** [comment]
bugs
bugs_organic

File: Bugzilla/Report/InactiveUser.pm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          OR  login_name LIKE '%\@bugzilla.org'
+        )
+        AND login_name <> 'nobody\@mozilla.com'
+        AND userid NOT IN ( SELECT DISTINCT who FROM bugs_activity )
+        AND userid NOT IN ( SELECT DISTINCT who FROM longdescs )
+        AND disabledtext = ''
+        ORDER BY userid
+    };
+
+    my $userids = $dbh->selectcol_arrayref($query);
```
Comment (dylanwh) [inline Bugzilla/Report/InactiveUser.pm]:
```suggestion
    return $dbh->selectcol_arrayref($query);
```

File: Bugzilla/Report/InactiveUser.pm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    # WARN(Dumper($userids));
+    sleep;
```
Comment (dylanwh) [inline Bugzilla/Report/InactiveUser.pm]:
```suggestion
```

File: Bugzilla/Report/InactiveUser.pm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    return $userids;
```
Comment (dylanwh) [inline Bugzilla/Report/InactiveUser.pm]:
```suggestion
```

File: Bugzilla/Report/InactiveUser.pm [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    #return Bugzilla::User->new_from_list($users);
```
Comment (dylanwh) [inline Bugzilla/Report/InactiveUser.pm]:
```suggestion
```

File: scripts/secbugsreport.pl [hunk 5]
```diff
@@ -22,6 +22,10 @@
 use Bugzilla::Mailer;
 use Bugzilla::Report::SecurityRisk;
 
+
```
Comment (dylanwh) [inline scripts/secbugsreport.pl]:
```suggestion
```

File: scripts/secbugsreport.pl [hunk 6]
```diff
@@ -22,6 +22,10 @@
...
+
```
Comment (dylanwh) [inline scripts/secbugsreport.pl]:
```suggestion
```

File: scripts/secbugsreport.pl [hunk 7]
```diff
@@ -22,6 +22,10 @@
...
+
```
Comment (dylanwh) [inline scripts/secbugsreport.pl]:
```suggestion
```

File: scripts/secbugsreport.pl [hunk 8]
```diff
@@ -22,6 +22,10 @@
...
+
```
Comment (dylanwh) [inline scripts/secbugsreport.pl]:
```suggestion
```

File: Bugzilla/Report/InactiveUser.pm [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+use Bugzilla::Types qw(User);
+use Type::Utils;
+use Types::Standard qw(:types);
+
+has 'dbh'     => (is => 'ro',   required => 1);
+has 'userids' => (is => 'lazy', isa      => ArrayRef [Int]);
+
+# TODO: double check that last_seen applies to api requests
+# TODO: ignore users with email in the badhosts
+# TODO: should we ignore .tld and .bugs?
```
Comment (dylanwh) [inline Bugzilla/Report/InactiveUser.pm]: Yes! Add those

**Comment by globau** [comment]
Fixed via https://github.com/mozilla-bteam/bmo/commit/b86f938b688cb72e1aee21080b2a6a627921865b instead.

**Pull request closed without merging**